src/test/java/org/isf/medicalsinventory/Tests.java

```diff
@@ -811,4 +828,86 @@ void testValidateMedicalInventory() throws Exception {
 		assertThat(medicalInventoryRows).hasSize(3);
 		medicalInventoryManager.validateMedicalInventoryRow(inventory, medicalInventoryRows);
 	}
+
+	@Test
+	void testValidateMedicalWardInventoryRow() throws Exception {
+		Ward ward = testWard.setup(false);
+		wardIoOperationRepository.saveAndFlush(ward);
+		MedicalInventory inventory = testMedicalWardInventory.setup(ward, false);
+		MedicalInventory savedInventory = medicalInventoryIoOperation.newMedicalInventory(inventory);
+		MedicalType medicalType = testMedicalType.setup(false);
+		medicalTypeIoOperationRepository.saveAndFlush(medicalType);
+		Medical medical = testMedical.setup(medicalType, false);
+		medicalsIoOperationRepository.saveAndFlush(medical);
+		Lot lot = testLot.setup(medical, false);
+		lotIoOperationRepository.saveAndFlush(lot);
+		MedicalInventoryRow medicalInventoryRow = testMedicalInventoryRow.setup(savedInventory, medical, lot, false);
+		int inventoryRowId = medicalInventoryRow.getId();
+		MedicalInventoryRow newMedicalInventoryRow = medicalInventoryRowManager.newMedicalInventoryRow(medicalInventoryRow);
+		assertThat(newMedicalInventoryRow).isNotNull();
+		List<MedicalInventoryRow> medicalInventoryRows = medicalInventoryRowManager.getMedicalInventoryRowByInventoryId(inventoryRowId);
+		assertThat(medicalInventoryRows).isNotEmpty();
+		assertThat(medicalInventoryRows).hasSize(1);
+		medicalInventoryManager.validateMedicalWardInventoryRow(savedInventory, medicalInventoryRows);
```

Comment:
this test is not actually testing anything if you don't make changes in the underlying records...
Please check the JUnit coverage to maximize the lines of code covered by the test:
![image](https://github.com/user-attachments/assets/a92c4dbc-9d14-427b-98af-8aee9fbee2e8)

Reply:
Please @ArnaudFofou verify also test coverage of your OP-1199.

Reply:
okay